app/templates/explore.hbs

```diff
@@ -7,6 +7,8 @@
     {{#each filteredEvents as |event|}}
       {{event-card event=event isWide=true shareEvent=(action 'shareEvent')}}
       <div class="ui hidden divider"></div>
+    {{else}}
+      <div class="ui disabled header">{{t 'No Events in this Category'}}</div>
```

Comment:
@ritikamotwani please change the helper text to *No events found for selected filters *